llvm/test/CodeGen/NVPTX/atomics-sm70.ll

```diff
@@ -0,0 +1,28 @@
+; RUN: llc < %s -march=nvptx -mcpu=sm_70 -mattr=+ptx63 | FileCheck %s
```

Comment:
For this test it might be convenient to autogenerate the checks with llvm/utils/update_test_checks.py

Reply:
Thanks for the suggestion. Done

Reply:
The ordering of the `ld.param.*`s relative to the `atom.*` instructions isn't relevant to this test, correct? If so, we may not want to include those `CHECK-NEXT:`s in the test.